packages/features/bookings/Booker/config.ts

```diff
@@ -224,7 +224,7 @@ export const extraDaysConfig = {
     tablet: 4,
   },
   [BookerLayouts.COLUMN_VIEW]: {
-    desktop: 4,
-    tablet: 2,
+    desktop: 5,
+    tablet: 3,
```

Comment:
@N-NeelPatel why there is change in column view. since the issue with week view..?

Reply:
The change in Header component fixes the issue in week view, but Header component is used by both week view and column view, and because of that it will show range of 4 days in desktop layout and 2 days in tablet layout.  something like this: 
![image](https://github.com/calcom/cal.com/assets/29038590/05c66f69-ac8e-4f5e-8195-1e4abd5a00d8)

But currently booker page shows 5 days for column view not 4. So I made changes for column view Desktop from 4 to 5 and for tablet from 2 to 3 to match current behavior. 

Furthermore, changes to the AvailableTimeSlots component was made to rectify the issue of displaying repetitive days and to show the appropriate days as intended.

Feel free to test locally to check the changes.

Reply:
Cool logically it's correct. Kindly take a look of this PR #9815